typed-racket-test/unit-tests/typecheck-tests.rkt

```diff
@@ -3126,14 +3127,14 @@
                   (-> Symbol #:b Symbol Symbol)))
            (define z (lambda (a #:b b) a))
            (z "y" #:b "y"))
-         #:ret (ret -String -bot-filter)
-         #:expected (ret -String -no-filter -no-obj)]
+         #:ret (ret -String -ff-propset)
+         #:expected (ret -String #f #f)]
```

Comment:
... huh? I'm either not seeing what your talking about or perhaps you miscounted? I see no differing numbers of arguments here across the diff.